modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -16,28 +16,28 @@ The Red Hat telco RAN DU {product-version} solution has been validated using the
 |Software version
 
 |Managed cluster version
-|4.16
+|4.17
```

Comment:
Same thought here about the attribute as above.

Reply:
See Aidan's previous, need to hardcode these.